book-worm-2010/book-worm-fe
# Feature/style animations

### What does this PR do? 
>Clearly and concisely describe the feature. 

Adds animation and styling to Login component and Celebration component with GSAP and SCSS.  When a user goes to the page, they are greeted with a login page.  Upon login, they are directed to the main page where they see their name on the screen and current books displayed (not connected yet).  The Celebration component will be triggered when a student clicks on the button to finish a book and an animation will render.

Styling was also added to make the x axis more readable on the reading progress display. Button functionality was added to the header, login, and logout buttons but pages are not fully connected yet with router.

### Is this a feature or Fix/Refactor?  

Feature

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.

There seems to be some missing functionality with rendering the current book list for a user from the server so the dummy data is still currently being displayed.  

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?

Login.js and Celebration.js.  Both components are currently rendering in App.js and Home.js

### How should this be tested?
>What steps need to be taken to confirm functionality? 

Run `npm start` in your terminal when in the project directory to open the app in your browser.  You will see the login page.  Sign in using Google auth and your google account.  Upon successful login, you will see the main display with your name in the header.  Right now, scroll down to the bottom of the page to see the Celebration component render.  Later, this will be changed to be viewed only on click for finishing a book.

![Screen Shot 2021-04-19 at 8 45 43 AM (2)](https://user-images.githubusercontent.com/67291333/115256610-9eb01d00-a0ec-11eb-9104-e45b837c07b5.png)


![Screen Shot 2021-04-19 at 8 45 27 AM (2)](https://user-images.githubusercontent.com/67291333/115256708-b5ef0a80-a0ec-11eb-9ef8-9aa10589d1ea.png)

### Steps To Take
>Any questions and possible steps moving forward.

Connect all of the routes and go through the user flow up through a finished book.
